server.js

```diff
@@ -66,3 +76,19 @@ function setCustomCacheControl(res, path) {
 
     res.setHeader('Cache-Control', 'public, max-age=0');
 }
+
+  // Error Handler
+  app.use((err, req, res) => {
+
+    if(err instanceof ApiError)
+        return res.status(err.status).send({ status:err.status, message: err.message });
+
+    res.status(500).send({ status: 500, message:  "Internal server Error"} );
+
+    console.error("*** Unhandled Exception:", err);
```

Comment:
its better to use the gaia logger, else we will never know when there is an error